tests/GridFS/StreamWrapperFunctionalTest.php

```diff
@@ -147,20 +148,19 @@ public function testWritableStreamSeek()
         $this->assertSame(-1, fseek($stream, 1, \SEEK_END));
     }
 
-    public function testWritableStreamStat()
+    public function testWritableStreamStatBeforeSaving()
     {
-        $currentTimestamp = time();
         $stream = $this->bucket->openUploadStream('filename', ['chunkSizeBytes' => 1024]);
 
         $stat = fstat($stream);
         $this->assertSame(0100222, $stat[2]);
         $this->assertSame(0100222, $stat['mode']);
         $this->assertSame(0, $stat[7]);
         $this->assertSame(0, $stat['size']);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat[9]);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat['mtime']);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat[10]);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat['ctime']);
+        $this->assertGreaterThanOrEqual(0, $stat[9]);
+        $this->assertGreaterThanOrEqual(0, $stat['mtime']);
+        $this->assertGreaterThanOrEqual(0, $stat[10]);
+        $this->assertGreaterThanOrEqual(0, $stat['ctime']);
```

Comment:
These should always default to `0`, so I think we can use `$this->assertSame(0, ...)` here.